cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -1252,6 +1252,7 @@
     public static final String SERVICE_PREFIX = "service.";
     public static final String FLOW_PREFIX = "flow.";
     public static final int RUN_RECORD_UPDATE_RETRY_DELAY_SECS = 5;
+    public static final int CREATE_LOCAL_DATASET_RETRY_DELAY_SECS = 5;
```

Comment:
I think similar retry logic would be required for deletion of the local datasets too. So we can rename this variable to the `LOCAL_DATASET_OPERATION_RETRY_DELAY_SECS`.